scripts/sync-react.js

```diff
@@ -343,13 +389,51 @@ Or run this command again without the --no-install flag to do both automatically
     )
   }
 
-  await fsp.writeFile(path.join(cwd, '.github/.react-version'), newVersionStr)
-
   if (errors.length) {
     // eslint-disable-next-line no-undef -- Defined in Node.js
     throw new AggregateError(errors)
   }
 
+  if (createPull) {
+    const github = await import('@actions/github')
+    const octokit = github.getOctokit(githubToken)
+    const branchName = `update/react/${newSha}-${newDateString}`
+
+    const pullRequest = await octokit.rest.pulls.create({
+      owner: repoOwner,
+      repo: repoName,
+      head: branchName,
+      base: 'main',
```

Comment:
Should this be 
```suggestion
      base: 'canary',
```
?